.ci/Jenkinsfile

```diff
@@ -295,7 +295,14 @@ def withCloudTestEnv(Closure body) {
   }
   // Masking
   withEnvMask(vars: maskedVars) {
-    body()
+    // Env variables to be populated as tag/labels for the cloud resources
+    // NOTE: values can contain only lowercase letters, numeric characters, underscores, and dashes
+    withEnv(["TF_VAR_CREATED_DATE_TIME=${getCurrentBuildTime().toLowerCase().replaceAll(':', '_')}",
+             "TF_VAR_CREATED_DATE=${new Date().format('yyyy-MM-dd')}",
+             "TF_VAR_BRANCH_NAME=${env.BRANCH_NAME.toLowerCase()}",
+             "TF_VAR_ENVIRONMENT=ci", "TF_VAR_REPO=${env.REPO.toLowerCase()}"]) {
```

Comment:
Can we move them to environment variables at the top of Jenkinsfile as we used to do?

Reply:
Sure, I intended to use them closed to the consumer to help with identifying what step will use it, but since `TF_VAR` is in place, I think that's not an issue